### PR TITLE
Enhancement/add tests markers

### PR DESCRIPTION
**Pull request by zbirobin**

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sweng/eventmanager/ui/EventMapFragmentTest.java [hunk 1]
```diff
@@ -37,9 +46,35 @@ public void eventMapTest() {
...
+            int x = marker.getBounds().centerX();
+            int y = marker.getBounds().centerY();
+            device.click(x, y-DIFFERENCE_TO_SNIPPET);
+        } catch (UiObjectNotFoundException e) {
+            e.printStackTrace();
+            fail("Fail on finding the marker");
+        }
+
+        //TODO: Test the different components of the map
+        /*// Clicks on the zoom in
```
Comment (Fnux) [inline app/src/androidTest/java/ch/epfl/sweng/eventmanager/ui/EventMapFragmentTest.java]: Why is there a commented block here?

**Review by Fnux — COMMENTED** [review]
Since the CI fails, this branch doesn't seem ready to be merged.

**Comment by Fnux** [comment]
Please also provide a meaningful description or a link to an existing issue.

**Comment by zyuiop** [comment]
What do you want to do with this branch? Please close the PR if it's not relevant anymore.

**Comment by zbirobin** [comment]
I tried several ways of testing the map, but each time, Travis doesn't support the tests. My point of view is that Travis can't display the map, so every test that checks the differents components of the map doesn't work. I have tried with UIAutomator and ideas of test inspired by the group named "Spaghettis coders".
https://codeclimate.com/github/Susanfe/sdp-event-management/app/src/main/java/ch/epfl/sweng/eventmanager/ui/eventShowcase/fragments/EventMapFragment.java/source

**Pull request closed without merging**